src/docfx/lib/moniker/MonikerDefinitionModel.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.Docs.Build
+{
+    internal class MonikerDefinitionModel
+    {
+        public List<Moniker> Monikers { get; } = new List<Moniker>();
+    }
```

Comment:
how about the DefinitionModel is just a list of array of Monikers?

Reply:
keep the Scalability just like `XrefMapModel`?

---
In reply to: [230978447](https://github.com/dotnet/docfx/pull/3575#discussion_r230978447) [](ancestors = 230978447)